interpreter_book/playground3/monkey/evaluator/evaluator_add_test.go

```diff
@@ -19,20 +19,20 @@ func TestArityCallExpressions(t *testing.T) {
 		errmsg string
 	}{
 		// to be specified: error msg
-		{"let truther = fn(x){false}; truther()", true, "not enough"},
-		{"let id = fn(x){x}; id()", true, "not enough"},
-		{"let add = fn(x,y){x+y}; add(2)", true, "not enough"},
+		{"let truther = fn(x){false}; truther()", true, "Not enough arguments!"},
+		{"let id = fn(x){x}; id()", true, "Not enough arguments!"},
+		{"let add = fn(x,y){x+y}; add(2)", true, "Not enough arguments!"},
```

Comment:
With these tests, I wanted to document what went wrong in the interpreter without deciding on a specific correct specification. So, the first step was for us to decide as a group what exactly we expect the interpreter to do in the face of the particular test cases. This was done here.